src/checker/hunspell.rs

```diff
@@ -243,6 +270,13 @@ fn obtain_suggestions<'s>(
             .filter(|x| x.len() > 1) // single char suggestions tend to be useless
             .collect::<Vec<_>>();
 
+        // strings made of vulgar fraction or emoji
+        if vulgar_fraction_or_emoji(word.clone()) {
+            trace!(target: "quirks", "Found emoji or vulgar fraction character, treating {} as ok", &word);
+            return;
```

Comment:
I can imagine, that cases of type `${fraction}-yada` would be ok as well, yet `yada` should potentially be checked.

Reply:
I am not sure what you mean with yada, is this a test case to be added?

Reply:
`yada` is just an example of an arbitrary suffix, so we could have constructs `½-baked` or `½-past` which might or might not be acceptable, but I guess that could be handled with regular hunspell quirks as well. Now that you found the solution using regex all of this could be done via regex, but I would prefer this to be a simple `on`/`off` configuration flag, rather than having people dig through this or accept false-positives.

So given that, ignore my comment above.